test/integration/client-side-encryption/client_side_encryption.spec.test.js

```diff
@@ -55,38 +57,18 @@ const skippedAuthTests = [
 const SKIPPED_TESTS = new Set(isAuthEnabled ? skippedAuthTests : []);
 
 describe('Client Side Encryption', function () {
-  // TODO: Replace this with using the filter once the filter works on describe blocks
-  const skipTests = process.env.CSFLE_KMS_PROVIDERS == null;
-  if (skipTests) {
-    // console.log('skipping Client Side Encryption Spec tests due to lack of AWS credentials');
-    return;
-  }
-
-  try {
-    require('mongodb-client-encryption');
-  } catch (e) {
-    console.error(
-      'skipping Client Side Encryption Spec tests due to inability to load mongodb-client-encryption'
-    );
-    return;
-  }
-
   const testContext = new TestRunnerContext();
+  testContext.requiresCSFLE = true;
   const testSuites = gatherTestSuites(
     path.join(__dirname, '../../spec/client-side-encryption/tests')
   );
-
   after(() => testContext.teardown());
   before(function () {
     return testContext.setup(this.configuration);
   });
 
   generateTopologyTests(testSuites, testContext, spec => {
-    return (
-      !spec.description.match(/type=symbol/) &&
-      !spec.description.match(/maxWireVersion < 8/) &&
-      !spec.description.match(/Count with deterministic encryption/) &&
-      !SKIPPED_TESTS.has(spec.description)
-    );
+    // TODO(NODE-xxxx): Investigate csfle test "operation fails with maxWireVersion < 8"
```

Comment:
Is there a valid Jira ticket for this TODO?

Reply:
NODE-4006